preprocess.py

```diff
@@ -0,0 +1,80 @@
+# from google.cloud import vision
```

Comment:
Add a comment about what is this doing.

Reply:
comment on what?

Reply:
The whole file. Preprocess is  too generic.